### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by spiekos**
Increase readability and include instructions for creating a private instance of the graph.

### CODE REVIEW

File: gke/README.md [hunk 1]
```diff
@@ -6,7 +6,19 @@ Instructions of the initial setup of the GCP project and GKE clusters for deploy
 
 * [Create a Google Cloud Project](https://cloud.google.com/resource-manager/docs/creating-managing-projects). If working on an existing project, you should have owner/editor role to perform the tasks.
 
-* Make a copy of the `config.yaml.tpl` as `config.yaml`.
+* Verify that yq version 3 is installed
```
Comment (shifucun) [inline gke/README.md:9]: Can you added a item like this:

* Install [yq](https://mikefarah.gitbook.io/yq/v/v3.x/). Make sure the version is 3.x
```bash
yq --version
```

File: gke/README.md [hunk 2]
```diff
@@ -55,8 +67,16 @@ Instructions of the initial setup of the GCP project and GKE clusters for deploy
 
   * If you see authentication errors, need to contact DataCommons team to complete some of the role binding operations as the service account need access to the Cloud Bigtable and Big Query.
 
+* [Optional]: If creating a private instance of the graph powered by the Data Commons Mixer complete the following steps:
```
Comment (shifucun) [inline gke/README.md:70]: This doc is meant to setup for the GKE instance.

I think we need a separate guide for starting a private mixer. Do you want to start a new doc under /docs/private_mixer.md and move this part there.

File: gke/README.md (hunk 1 above)
Comment (spiekos) [reply gke/README.md]: changed

File: gke/README.md (hunk 2 above)
Comment (spiekos) [reply gke/README.md]: Removed the optional steps for the private instance to be included in a separate document

**Review by spiekos — COMMENTED** [review]
changed yq instructions and removed optional instructions for private instance to include in separate documentation

**Review by shifucun — APPROVED** [review]
thanks for the update!

**Pull request merged**